src/kbmod/search/layered_image.cpp

```diff
@@ -306,6 +316,9 @@ static void layered_image_bindings(py::module& m) {
 
     py::class_<li>(m, "LayeredImage", pydocs::DOC_LayeredImage)
             .def(py::init<const ri&, const ri&, const ri&, pf&>())
+            .def(py::init<search::Image&, search::Image&, search::Image&, pf&, double>(),
+                 py::arg("sci").noconvert(true), py::arg("var").noconvert(true),
+                 py::arg("msk").noconvert(true), py::arg("psf"), py::arg("obs_time"))
```

Comment:
What do these `noconvert`s do? Just curious, it's unfamiliar.

Reply:
I don't know all the details. But they are needed for passing numpy arrays of float32s into Eigen matrices in C++. I'm not sure if it prevents it from converting to a float64 or into a list of lists or something else.